### DISCUSSION TITLE
Final internship application

### DISCUSSION THREAD

**Discussion by tristanlatr** [LesFruitsDefendus/saskatoon-ng#253] [Outreachy]
Hello everyone, 

We received several strong initial applications, but no final applications yet. 

I encourage you to submit your final application by following the link below:

https://www.outreachy.org/outreachy-may-2022-internship-round/communities/les-fruits-defendus/getting-saskatoon-ng-ready/contributions/

Thanks a lot for all of your contributions,

Talk to you later,

**Comment by farhanmasud** [comment]
Hi @tristanlatr in the final application we are supposed to fill up a timeline based on the work to be done during the internship. The issues that were listed in the outreachy page and were supposed to be worked on the internship period, those are already being worked on and many of them will be completed within the contribution period.

Could you please help us make an outline for the internship tasks so that we could make an outline of the timeline and create the final application?

**Reply by tristanlatr** [reply]
> The issues that were listed in the outreachy page and were supposed to be worked on the internship period, those are already being worked on and many of them will be completed within the contribution period.

This is true… I’ll review the issues listed in the description and update it accordingly. I’ll keep you posted here. I’ll try to do that today.

Thanks,

**Comment by tristanlatr** [comment]
Hi @farhanmasud, and others,

Here is the updated description of the project, as you can see there is a lot of flexibility in the internship objectives:

The intern tasks will be to develop new features and improve code quality of Saskatoon.

You will work on hight priority tasks as well as long-term development features. 

High priority issues are mark with the label "[high priority](https://github.com/LesFruitsDefendus/saskatoon-ng/issues?q=is%3Aissue+is%3Aopen+label%3A%22high+priority%22)".

Tasks that are proposed as objectives for the internship are marked with the label "[proposed internship task](https://github.com/LesFruitsDefendus/saskatoon-ng/labels/proposed%20internship%20task)". The objectives will be talk about at be beginning of the internship period.

Talk to you later,

**Reply by farhanmasud** [reply]
@tristanlatr I had one question about the new description.

>The intern tasks will be to develop new features and improve code quality of Saskatoon.

Does the "develop new features" refer to the issues under ["proposed internship tasks"](https://github.com/LesFruitsDefendus/saskatoon-ng/labels/proposed%20internship%20task) label?

**Reply by tristanlatr** [reply]
It's more the long-term development features that refers to this label. Some simple features would be good to have this season :)

**Reply by farhanmasud** [reply]
Got it. Thanks!

**Comment by victorphoenix3** [comment]
Hi @tristanlatr,  Can you please share your email-id, so that I can share my proposal with you for your feedback and fix it accordingly before submitting it on the outreachy portal?

**Reply by tristanlatr** [reply]
Hi @victorphoenix3, you can email us at info@lesfruitsdefendus.org, I’ll be checking this email more regularly in the coming days.

Talk to you later,
